download.md

```diff
@@ -9,7 +9,7 @@ no-wrapper: true
 <div class="row margin-bottom-xl">
     <div class="col-sm-12">
         <p class="lead">
-            <span class="glyphicon glyphicon-download"/>
+            <span class="glyphicon glyphicon-download"></span>
```

Comment:
Because self-closing spans are illegal HTML, and this caused extreme issues in browsers running strict mode.